Source/WebKit/UIProcess/WebPageProxy.h

```diff
@@ -29,6 +29,7 @@
 // Use forward declarations and WebPageProxyInternals.h instead.
 #include "APIObject.h"
 #include "MessageReceiver.h"
+#include "WebProcessProxy.h"
```

Comment:
This should stay a forward declaration.  If a function doesn't compile, move it to WebPageProxy.cpp.